packages/wallet/src/services/TipTracker.ts

```diff
@@ -62,20 +66,34 @@ export class TipTracker extends PersistentDocumentTrackerSubject<Cardano.Tip> {
           connectionStatus$
         ]).pipe(
           // Throttle syncing by interval, cancel ongoing request on external trigger
+          tap(([isSettled, connectionStatus]) => {
+            if (connectionStatus === ConnectionStatus.down) {
+              // REVIEW: slightly redundant since connection debug messages are already recorded separately
+              logger.debug('Connection down. Skipping request');
+            } else {
+              logger.debug(isSettled === null ? 'Initial request' : 'Start request');
+            }
+          }),
           exhaustMap(([_, connectionStatus]) =>
             connectionStatus === ConnectionStatus.down ? EMPTY : provider$.pipe(takeUntil(externalTrigger$))
           ),
-          distinctUntilChanged(tipEquals)
+          distinctUntilChanged(tipEquals),
+          tap((tip) => logger.debug('Fetched new tip', tip))
```

Comment:
Is it ok to log the whole tip object? Alternatively I could log only it's hash

Reply:
I think it's ok

Reply:
The whole tip is beneficial as the combination of slot number and hash can detect a fork.